docs/libblockdev-docs.xml.in

```diff
@@ -54,6 +54,53 @@
       Python (under the <filename class="directory">src/python</filename> directory).
     </para>
   </chapter>
+
+  <chapter>
+    <title>Testing libblockdev</title>
+    <para>
+      libblockdev's test suite is written using the standard unittest.TestCase framework
+      in Python. Tests are separated in modules, usually one per libblockdev plugin - e.g.
+      BTRFS, LVM, Crypto. There are one or more base classes in each module used to
+      setup the environment and perform some testing. More specific test scenarios
+      inherit from these base classes.
+    </para>
+
+    <para>
+      Before running the tests you have to prepare your system so that libblockdev
+      can be built from source.
+    </para>
+
+    <para>
+      Install all build requirements. On Fedora this can be done with
+      <screen><userinput>
+cat dist/libblockdev.spec| grep BuildRequires: | cut -f2 -d: | cut -f2 -d' ' | xargs dnf -y install
```

Comment:
This should be ``cat dist/libblockdev.spec.in...``, the .spec file is produced by ``./configure`` and doesn't exist before running them.